plugin/unique-ckpt/unique-ckpt.cpp

```diff
@@ -16,7 +16,13 @@ extern "C" int dmtcp_unique_ckpt_enabled(void)
 
 void updateCkptDir()
 {
-  const char *ckptDir = dmtcp_get_ckpt_dir();
+  const char *ckptDir = NULL; 
+  string globalCkptDir = dmtcp_get_global_ckpt_dir();
+  if(globalCkptDir.empty())
+    ckptDir = dmtcp_get_ckpt_dir();
```

Comment:
Nevermind, I figure it out. While the former queries the coordinator, the latter returns the directory that was set at launch time, if I'm reading this correctly.